test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1753,7 +1801,6 @@ class test_ucp_sockaddr_protocols_err : public test_ucp_sockaddr_protocols {
                                                         variants & RECV_STOP);
     }
 
-protected:
```

Comment:
why removed?

Reply:
do we need to declare 'protected' for functions and vars separately?

Reply:
ok